checker/src/main/java/org/checkerframework/checker/calledmethods/framework/FrameworkSupport.java

```diff
@@ -0,0 +1,50 @@
+package org.checkerframework.checker.calledmethods.framework;
+
+import com.sun.source.tree.NewClassTree;
+import javax.lang.model.element.ExecutableElement;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+
+/** add CalledMethods annotations to code generated by framework */
+public interface FrameworkSupport {
+
+    /**
+     * Allows for supporting a framework's {@code toBuilder} routine. Typically, the returned
+     * Builder has had all of its required setters invoked. So, the framework support can add a
+     * {@link org.checkerframework.checker.calledmethods.qual.CalledMethods} annotation capturing
+     * this fact.
+     *
+     * @param t a method that is possibly the {@code toBuilder} method for a type that has an
+     *     associated builder. An implementation must check that this is actually the {@code
+     *     toBuilder} method for the corresponding framework.
+     */
+    void handlePossibleToBuilder(AnnotatedTypeMirror.AnnotatedExecutableType t);
+
+    /**
+     * For {@code build} methods on {@code Builder} types, the framework support should determine
```

Comment:
I was confused by this comment.  I now think "should determine" may describe what an implementation of this method should do, but that was not clear.  On rereading it makes sense, but please rephrase it to make sense the first time through.  This comment applies to all of the methods in this interface.